expo/snack
# [website] Refactor (generalize) modal showing

# Why

Consolidate code by showing modals using a single function.

# How

- Add generalised method for showing modals

# Test Plan

- Verified locally and confirmed the following modals work as expected
  - Save
  - Run on device
  - Show embed code
  - Shortcuts
  - Import files
  - Import repository
  - Import from production
  - Edit details
